### ISSUE TITLE
SlackNotifier doesn't accept direct-messages

### ISSUE THREAD

**Issue by saidimu** [SSENSE/node-framework#14]
From [this line](https://github.com/SSENSE/node-framework/blob/cf00d931dda8ff6043929875f23c0e7de31f99df/src/http/SlackNotifier.ts#L26) the `defaultChannel` is restricted to only be a Slack channel but Slack also allows direct messages by specifying `@username`.

**Comment by RemyJeancolas** [comment]
You're totally right, as we didn't need it, I restricted the library to send on channels only, but it was a useless limitation.
I just pushed a new version `v0.3.1` which will allow you to do this, it's of course backwards compatible.
Thanks for the feedback :+1:
